e2e-tests/content-strings/pages/index.js

```diff
@@ -111,6 +112,35 @@ const PAGE_NOT_FOUND_PAGE = {
   },
 };
 
+const CONTACT_US_PAGE = {
+  PAGE_TITLE: 'Contact us',
+  WHO_TO_CONTACT: 'Who you need to contact depends on your type of query.',
+  GENERAL_ENQUIRIES: {
+    HEADING: 'General enquiries or questions about UK Export Finance products',
+    TELEPHONE: {
+      PREFIX: 'Telephone: ',
+      VALUE: CONTACT_DETAILS.TELEPHONE,
+    },
+    EMAIL: {
+      PREFIX: 'Email: ',
+      VALUE: CONTACT_DETAILS.EMAIL.CUSTOMER_SERVICE,
+    },
+    OPENING_TIMES: 'Opening times: Monday to Friday, 9am to 5pm (excluding public holidays)',
+    CALL_CHARGES: {
+      TEXT: 'Call charges',
+      HREF: LINKS.EXTERNAL.CALL_CHARGES,
+    },
+  },
+  APPLICATION_ENQUIRES: {
+    HEADING: "If you have a question about an application you've submitted",
+    EMAIL: {
+      PREFIX: 'Email: ',
+      VALUE: CONTACT_DETAILS.EMAIL.UNDERWRITING,
+    },
+    QUOTE: 'Quote your application reference number',
+  },
+};
+
```

Comment:
The contact page is going to replace the footer "contact" content - this is in another ticket I believe? We can delete some contact content strings (for the footer) when we get to this

Reply:
yes, that is my plan in the next PR